german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/gruessen_freundlich.md

```diff
@@ -1,4 +1,6 @@
 "Hallo, ich heiße Bernd und bin schiffbrüchig". Die beiden drehen sich schlagartig
 in Ihre Richtung und starren Sie mit großen Augen an. Entsetzen ist in
 ihr Gesicht geschrieben. Gerade als Sie ...
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.